beard.js

```diff
@@ -47,17 +50,17 @@ class Beard {
     }
   }
 
-  asset(p, path) {
-    const absolutePath = resolvePath(p, path);
-    return this.opts.asset(absolutePath) || absolutePath;
+  callback(name, path, parentPath, data) {
+    const resolvedPath = resolvePath(path, parentPath);
+    return this.opts.callbacks[name](resolvedPath, data);
   }
 
   render(path, data = {}) {
     const context = {
       globals: {},
       locals: [data],
       compiled: this.compiled.bind(this),
-      asset: this.asset.bind(this)
+      callback: this.callback.bind(this)
```

Comment:
@joeosburn not sure I love `callbacks` as the name for these. What about something like `tags`?